spring-boot-starter/src/main/java/org/togglz/spring/boot/autoconfigure/TogglzEndpoint.java

```diff
@@ -54,6 +54,17 @@ public TogglzEndpoint(FeatureManager featureManager) {
         return features;
     }
 
+    public Optional<Feature> get(String name) {
+        return featureManager.getFeatures().stream().filter(f -> name.equals(f.name())).findFirst();
```

Comment:
TogglzEndpoint.java:[58,63] lambda expressions are not supported in -source 1.7
https://travis-ci.org/togglz/togglz/jobs/228788533